test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -3227,4 +3227,120 @@ describe('v1.0.0 to v2.0.0', function () {
             });
         });
     });
+
+    describe('handleBodyOptions', function () {
```

Comment:
```suggestion
    describe('dataOptions', function () {
```

Test using both `convert` and `convertSingle`. Refer `protocolProfileBehavior ` test.